documentation/install_from_packages.markdown

```diff
@@ -1,16 +1,25 @@
 ---
 layout: default
-title: "Puppet Server: Installing From Packages"
+title: "Install Puppet Server"
 canonical: "/puppetserver/latest/install_from_packages.html"
 ---
 
 [repodocs]: https://puppet.com/docs/puppet/latest/puppet_platform.html
 
-Puppet Server is configured to use 2 GB of RAM by default. If you're just testing an installation on a Virtual Machine, this much memory is not necessary. To change the memory allocation, see [Memory Allocation](#memory-allocation).
+Puppet provides official packages that install Puppet Server 6.0 and all of its prerequisites on x86_64 architectures for the following platforms, as part of [Puppet Platform][repodocs].
```

Comment:
This should probably just be "6" not "6.0", since the 6.x stream has many other Y releases that are also installed this way.

Reply:
is `repodocs` meant to be a place holder for something?